pinconfigs/gd32e103xx.yml

```diff
@@ -0,0 +1,790 @@
+  PB6:
+    pincodes: [C, V, R, T]
+    afs: ['I2C0_SCL', 'TIMER3_CH0'] 
+  PB7:
+    pincodes: [C, V, R, T]
+    afs: ['TIMER3_CH1', 'EXMC_NL(NADV)', 'I2C0_SDA'] 
+  PB8:
+    pincodes: [C, V, R]
+    afs: ['TIMER9_CH0', 'TIMER3_CH2'] 
+  PB9:
+    pincodes: [C, V, R]
+    afs: ['TIMER10_CH0', 'TIMER3_CH3'] 
+  PC0:
+    pincodes: [V, R]
+    afs: ['ADC01_IN10'] 
+  PC1:
+    pincodes: [V, R]
+    afs: ['ADC01_IN11'] 
+  PC10:
+    pincodes: [V, R]
+    afs: ['UART3_TX'] 
+  PC11:
+    pincodes: [V, R]
+    afs: ['UART3_RX'] 
+  PC12:
+    pincodes: [V, R]
+    afs: ['UART4_TX'] 
+  PC13:
+    pincodes: [C, V, R]
+    afs: ['RTC_TAMPER'] 
+  PC14:
+    pincodes: [C, V, R]
+    afs: ['OSC32IN'] 
+  PC15:
+    pincodes: [C, V, R]
+    afs: ['OSC32OUT'] 
+  PC2:
+    pincodes: [V, R]
+    afs: ['ADC01_IN12'] 
+  PC3:
+    pincodes: [V, R]
+    afs: ['ADC01_IN13'] 
+  PC4:
+    pincodes: [V, R]
+    afs: ['ADC01_IN14'] 
+  PC5:
+    pincodes: [V, R]
+    afs: ['ADC01_IN15'] 
+  PC6:
+    pincodes: [V, R]
+    afs: ['TIMER7_CH0', 'I2S1_MCK'] 
+  PC7:
+    pincodes: [V, R]
+    afs: ['TIMER7_CH1', 'I2S2_MCK'] 
+  PC8:
+    pincodes: [V, R]
+    afs: ['TIMER7_CH2'] 
+  PC9:
+    pincodes: [V, R]
+    afs: ['TIMER7_CH3'] 
+  PD0:
+    pincodes: [C, V, R, T]
+    afs: ['EXMC_D2'] 
+  PD1:
+    pincodes: [C, V, R, T]
+    afs: ['EXMC_D3'] 
+  PD10:
+    pincodes: [V]
+    afs: ['EXMC_D15'] 
+  PD11:
+    pincodes: [V]
+    afs: ['EXMC_A16'] 
+  PD12:
+    pincodes: [V]
+    afs: ['EXMC_A17'] 
+  PD13:
+    pincodes: [V]
+    afs: ['EXMC_A18'] 
+  PD14:
+    pincodes: [V]
+    afs: ['EXMC_D0'] 
+  PD15:
+    pincodes: [V]
+    afs: ['EXMC_D1'] 
+  PD2:
+    pincodes: [V, R]
+    afs: ['UART4_RX', 'TIMER2_ETI'] 
+  PD3:
+    pincodes: [V]
+    afs: ['EXMC_CLK'] 
+  PD4:
+    pincodes: [V]
+    afs: ['EXMC_NOE'] 
+  PD5:
+    pincodes: [V]
+    afs: ['EXMC_NWE'] 
+  PD6:
+    pincodes: [V]
+    afs: ['EXMC_NWAIT'] 
+  PD7:
+    pincodes: [V]
+    afs: ['EXMC_NE0'] 
+  PD8:
+    pincodes: [V]
+    afs: ['EXMC_D13'] 
+  PD9:
+    pincodes: [V]
+    afs: ['EXMC_D14'] 
+  PE0:
+    pincodes: [V]
+    afs: ['EXMC_NBL0', 'TIMER3_ETI'] 
+  PE1:
+    pincodes: [V]
+    afs: ['EXMC_NBL1'] 
+  PE10:
+    pincodes: [V]
+    afs: ['EXMC_D7'] 
+  PE11:
+    pincodes: [V]
+    afs: ['EXMC_D8'] 
+  PE12:
+    pincodes: [V]
+    afs: ['EXMC_D9'] 
+  PE13:
+    pincodes: [V]
+    afs: ['EXMC_D10'] 
+  PE14:
+    pincodes: [V]
+    afs: ['EXMC_D11'] 
+  PE15:
+    pincodes: [V]
+    afs: ['EXMC_D12'] 
+  PE2:
+    pincodes: [V]
+    afs: ['EXMC_A23', 'TRACECK'] 
+  PE3:
+    pincodes: [V]
+    afs: ['EXMC_A19', 'TRACED0'] 
+  PE4:
+    pincodes: [V]
+    afs: ['EXMC_A20', 'TRACED1'] 
+  PE5:
+    pincodes: [V]
+    afs: ['TRACED2', 'EXMC_A21'] 
+  PE6:
+    pincodes: [V]
+    afs: ['TRACED3', 'EXMC_A22'] 
+  PE7:
+    pincodes: [V]
+    afs: ['EXMC_D4'] 
+  PE8:
+    pincodes: [V]
+    afs: ['EXMC_D5'] 
+  PE9:
+    pincodes: [V]
+    afs: ['EXMC_D6'] 
```

Comment:
looks like you're missing pins: what about PE10..PE15?

Reply:
PE10..PE15 at line 649-666.
I'll change the pin order.